### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by AdiSaranga**
In this branch I connected the view to the model, 
there are also some UI changes,
I added cuisine data and restaurants to seed.rb

### CODE REVIEW

File: app/javascript/packs/weeatApp/components/restaurantsIcon.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+class RestaurantIcon extends React.Component {
+  render() {
+    const defaultIcon = 'F';
+
+    const cuisineTypeToIcon = {
```
Comment (isaac-fisher) [inline app/javascript/packs/weeatApp/components/restaurantsIcon.jsx]: This should be inside a table on the server side. If someone wants to add a cuisine he doesn't need to made code changes that way and all his changes are in the same place.

File: app/javascript/packs/weeatApp/assets/stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,148 @@
+@font-face {
```
[the remark this answers is not in the document]
Comment (isaac-fisher) [reply app/javascript/packs/weeatApp/assets/stylesheet.css]: ok, got it, you have probably committed to the same branch as the previous PR.
notice that a PR is updating for every commit to its branch so you should add a new branch for any PR.

File: app/controllers/restaurants_controller.rb [hunk 3]
```diff
@@ -4,12 +4,13 @@ class RestaurantsController < ApplicationController
...
   # GET /restaurants.json
   def index
-    @restaurants = Restaurant.all
+    render json: Restaurant.all
   end
 
   # GET /restaurants/1
   # GET /restaurants/1.json
   def show
+    render json: @restaurant
```
[the remark this answers is not in the document]
Comment (AdiSaranga) [reply app/controllers/restaurants_controller.rb]: Done, will be part of next PR

File: app/javascript/packs/weeatApp/components/restaurantsIcon.jsx (hunk 1 above)
Comment (AdiSaranga) [reply app/javascript/packs/weeatApp/components/restaurantsIcon.jsx]: moved to DB, will be part of next PR

**Pull request merged**